### PR TITLE
Lower minumum batch size to 1 for background updates

### PR DESCRIPTION
**Pull request by babolivier**
See https://github.com/matrix-org/synapse/issues/11401 for context

### CODE REVIEW

File: changelog.d/11422.bugfix [hunk 1]
```diff
@@ -0,0 +1 @@
+Improve performance of various background updates related to device inboxes.
```
Comment (richvdh) [inline changelog.d/11422.bugfix:1]:
```suggestion
Improve performance of various background database schema updates.
```

(it's not *just* the device inbox stuff we're targetting here)

**Review by richvdh — APPROVED** [review]
lgtm

File: changelog.d/11422.bugfix (hunk 1 above)
Comment (babolivier) [reply changelog.d/11422.bugfix:1]: Yeah but the idea is to merge this changelog with the ones from the other PRs that sprouted out of https://github.com/matrix-org/synapse/issues/11401

Comment (richvdh) [reply changelog.d/11422.bugfix:1]: update those too :)

Comment (babolivier) [reply changelog.d/11422.bugfix:1]: (ah I've just now seen your comment on #11421)

**Pull request merged**